operator/v1/types_ingress.go

```diff
@@ -55,11 +55,8 @@ type IngressControllerSpec struct {
 	// +optional
 	Domain string `json:"domain,omitempty"`
 
-	// replicas is the desired number of ingress controller replicas. If unset,
-	// defaults to 2.
-	//
-	// +optional
-	Replicas *int32 `json:"replicas,omitempty"`
+	// replicas is the desired number of ingress controller replicas.
+	Replicas int32 `json:"replicas"`
```

Comment:
@smarterclayton is there a reason you wanted this to be a pointer? Seems like it has to be a value type for scale subresource to work.

Reply:
Every workload object should have pointer to replicas - see RS, STS, Deployments, etc.

Reply:
Why are you different than deployments which have a scale resource?  If you say "scale X" of course you have specify X.

Reply:
I see that now... must something we're missing wrt scale